extensions/wikia/ThemeDesigner/UploadFaviconFromFile.class.php

```diff
@@ -7,48 +8,48 @@ class UploadFaviconFromFile extends UploadFromFile {
 	// TODO: Are these valid error codes?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)